Edit USUM Wormhole guide (#127)
zaksabeast/PokemonRNGGuides: guides/Ultra Sun and Ultra Moon/Stationary Wormhole RNG.md

- [3DSRNGTool](https://github.com/wwwwwwzx/3DSRNGTool/releases)
- PCalc
  - [Ultra Sun/Ultra Moon](https://pokemonrng.com/downloads/pcalc/pcalc-usum.zip)

> - [3DSRNGTool README](https://github.com/wwwwwwzx/3DSRNGTool/blob/master/README.md#final-screen) - list of final screens before the Pokemon is generated

Reviewer: ```suggestion
- [3DSRNGTool README](https://github.com/wwwwwwzx/3DSRNGTool/blob/master/README.md#final-screen) - list of final screens before Pokemon are generated
```

> ## Step 1: Finding your seed

Reviewer 2: I'd add a note that it works for UBs too ?

So what happened?

This is okay and normal as each 3DS/save has its own delay that you will need to find. The delay is the time it takes from when you initiate the encounter and when the Pokemon actually gets generated by the game. Let's learn how to do that!

## Step 4: Finding your delay and hitting it

1. We can use the IVs, nature and PSV of the Pokemon we got to see what frame we actually hit. My target was 4973 and I hit 4969 (-4). So the delay I hit vs the default setting of the 3DSRNGTool is -4 or 152.

> 

Reviewer 2: Well, it's not exactly where I want this note to be but whatever : 

In the step 2) when the subject of repeating it, it'd be important to add a note for the Legendaries : Being precise and doing the same type of input (direction / pressure) will help to have a stable delay. The idea is that the last input is a step done by the stick. So not really accurate. So it's important to always do the same kind of input.
